_posts/2021-02-22-demo-day-8-contributing-psl.md

```diff
@@ -0,0 +1,40 @@
+---
+toc: true
+layout: post
+description: How to contribute to PSL projects
+categories: [demo-days, PSL, git, github]
+author: Jason DeBacker
+title: "Demo Day 7: How to contribute to PSL projects"
```

Comment:
Demo Day 8

Reply:
Thanks for the catch @MattHJensen - fixed this in commit below.